client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -229,7 +230,7 @@ export const VaccineCourseEditModal: FC<VaccineCourseEditModalProps> = ({
       }
       height={height}
       sx={{
-        width: 875,
+        width: 950,
```

Comment:
Pretty much the minimum value that doesn't require side-scrolling. Fortunately the modal still shrinks as the viewport narrows.

Reply:
I think this is ok, its only accessible on central server 👍